### PR TITLE
Stage split into packages.

### PR DESCRIPTION
**Pull request by jdhuntington**
# Draft PR

*See #9645 for context*

## Things to review
- staged packages live in `office-ui-fabric-react/src/packages/react-*`
- good-fences is part of the build in OUFR; as of this PR it validates that focus-zone-* takes only necessary dependencies
- examples are broken out into an `examples` package

## Next steps
- Get comments on this PR
- Rebase on `master` after getting feedback (there will be conflicts)
- commit rebase then merge, along with communication to contributors about change and expected upcoming changes
- continue to tease apart dependencies by using good fences
- once all dependencies are broken apart, automate splitting into actual separate packages

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9801)

### CODE REVIEW

File: packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md [hunk 1]
```diff
@@ -460,6 +460,18 @@ export class Button extends BaseComponent<IButtonProps, {}> {
     protected _skipComponentRefResolution: boolean;
 }
 
+// @public (undocumented)
+export const ButtonGlobalClassNames: {
```
Comment (Vitalius1) [inline packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md]: Not sure we want these surfaced? At least if we decide to do it should be in a consistent manner for all the controls

File: packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.doc.tsx [hunk 2]
```diff
@@ -1,25 +0,0 @@
-import * as React from 'react';
```
Comment (Vitalius1) [inline packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.doc.tsx]: Marking these file as deleted… does that mean we loose the git history of the file in its new place?

File: packages/office-ui-fabric-react/src/packages/react-commanding/components/Nav/Nav.doc.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
```
Comment (Vitalius1) [inline packages/office-ui-fabric-react/src/packages/react-commanding/components/Nav/Nav.doc.tsx]: I am guessing using VS code extensions like Git Lens that is displaying inline who last edited the line will be useless now 😞

**Comment by khmakoto** [comment]
@jdhuntington should we close this draft PR?

**Pull request closed without merging**